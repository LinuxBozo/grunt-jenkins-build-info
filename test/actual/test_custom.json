{
  "name": "grunt-jenkins-build-info",
  "version": "0.0.1"

}