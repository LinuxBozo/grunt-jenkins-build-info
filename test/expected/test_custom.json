{
  "name": "grunt-jenkins-build-info",
  "version": "0.0.1",
  "buildInfo": {
    "number": "foo",
    "tag": "tag",
    "url": "http://example.org",
    "svnRevision": "bar",
    "gitRevision": "baz",
    "gitBranch": "develop"
  }
}